config.py

```diff
@@ -581,7 +581,13 @@ def __init__(self) -> None:
 
     def __raw_get(self, key: str) -> Union[None, list, str, int]:
         res = self._settings.get(key)
-        if isinstance(res, list):
+        # On MacOS Catalina, with python.org python 3.9.2 any 'list'
+        # has type __NSCFArray so a simple `isinstance(res, list)` is
+        # False.  So, check it's not-None, and not the other types.
+        #
+        # If we can find where to import the definition of NSCFArray
+        # then we could possibly test against that.
+        if res and not isinstance(res, str) and not isinstance(res, int):
```

Comment:
```suggestion
        if res is not None and not isinstance(res, str) and not isinstance(res, int):
```
`if res` will be false if res is `[]`